### ISSUE TITLE
Moving Site

### ISSUE THREAD

**Issue by markwaters** [idno/Known#644]
I would like to move my site from one domain to another.
Is there a way to import the old export files somewhere ?
Is there a way to update all the posts to change the old URL to the new URL ?
Thanks!
##

**Comment by benwerd** [comment]
You should just be able to export the entities table and import it elsewhere - Known will cope. You'll also need to rename the upload directory (its first subdirectory is your hostname).

We have plans for a much easier method that uses a more OAuth-like flow.

**Comment by markwaters** [comment]
thanks @benwerd , that sounds nice and easy , I will give it a try in the next few days and report back.

**Comment by markwaters** [comment]
Also , how would I go about merging my old known 'test' site into my proper new known site ?

**Comment by benwerd** [comment]
Was this a hosted site? We can send you a data export.

**Comment by markwaters** [comment]
No , these are sites I am hosting locally.

**Comment by michielbdejong** [comment]
@benwerd wrote:
> Was this a hosted site? We can send you a data export.

Great! I do have two people want to migrate their *.withknown.com instance to me. So I will tell them to contact you and request a dump of their entities table and uploads folder. Would this be free of charge?

**Comment by benwerd** [comment]
Sorry I missed this - absolutely free of charge.

However, we're updating the export function to make this even easier. You'll just be able to get the complete files folder and SQL, and plug them straight in.

**Comment by benwerd** [comment]
This export change has now been completed.

**Comment by physcocode** [comment]
+1 @benwerd this saved my life *_*

**Comment by pierreozoux** [comment]
Can we close this issue?

**Comment by markwaters** [comment]
I recently tried switching my wifes site from one domain to another while using the same database , I also copied the Uploads/ directory to the new site , the post urls were correctly using the new domain but the photos were still being displayed from the old domain and thus broken.

**Comment by tinokremer** [comment]
Same problem here. Though I moved from my WithKnown hosted stream.tinokremer.nl to selfhosted. I imported the SQL dump and put the Uploads folder back in place. The odd thing is, the images do not show up indeed (404, not found), but the 'permalink' on the images itself is fine:

So as an example. One of the images has an img src of:
http://stream.tinokremer.nl/file/5ef996160feda74713657da2cc82b6b2/thumb.jpg
(not working)

And a link (a href) of:
http://stream.tinokremer.nl/file/7b4f1e1c51416c3e0263ddef23dadb32/IMG_20150728_203620.jpg

...Working just fine.

I also tried adding a new photo for testing purposes and that one has both links work fine.
